baseMdl/mdl.py

```diff
@@ -168,40 +62,17 @@ def import_model():
                     f"Can't read an empty file."
                 )
 
-    def watch_trained_model(self, mdl_q_table):
-        current_state = self.discretize(
-            self.env.reset(), self.lower_bounds, self.upper_bounds, self.buckets
-        )
-        total_reward = 0
-
-        print("Watching trained model...")
-        while True:
-            self.env.render()
-
-            action = np.argmax(mdl_q_table[current_state])
-
-            observation, reward, done = self.env.step(action)
-
-            new_state = self.discretize(
-                observation, self.lower_bounds, self.upper_bounds, self.buckets
-            )
-
-            total_reward += reward
-
-            current_state = new_state
-
-            if done:
-                break
-        print("Finished watching trained model...")
+    def watch_trained_model(self):
+        pass
 
-    def ask_to_save_model(mdl_q_table):
+    def ask_to_save_model(self):
         while True:
             save = input("Save model? [y/N]: ")
             if save in ("", "N", "n"):
                 return
             elif save in ("Y", "y"):
                 mdl_file = input("Enter model file name: ")
```

Comment:
Estuve probando y vi que el modelo se guarda en el directorio donde corres el script, por ejemplo si lo corres en /games, se guarda el modelo en /games. Estaba pensando que sería más conveniente que se guarden en un mismo directorio.

Esto se podría lograr agregando algo así:
```suggestion
                mdl_file = input("Enter model file name: ")
                # Gets the path to the savedModels directory
                models_dir = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), self.models_dir_name)
                if not os.path.exists(models_dir):
                    os.makedirs(models_dir)
                # Gets the path to the saved model file
                mdl_file = os.path.join(models_dir, mdl_file)
``` 
Haciendo eso, se guardan las q-tables en uno que se llama "savedModels" que está en el root del proyecto, y crea el directorio si es que no existe (para evitar que falle en dado caso).